### DISCUSSION TITLE
反馈

### DISCUSSION THREAD

**Discussion by Lorangeve** [Richasy/Bili.Uwp#101] [问答]
1. 打开更多评论,希望在右边栏
2. 亚克力很好看
3. 鼠标滚轮（触控板也是）只能左右滚动，不能上下滚动；上下滚动只能拖动滚动条

**Comment by Richasy** [comment]
滚动问题已在新版中修复。
评论位置的修改已经提上日程

**Comment by tm0210** [comment]
这个是云母材质，不是亚克力材质(=ó∀ò)o中秋快乐
